mappings/net/minecraft/command/ICommandManager.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS none/m net/minecraft/command/ICommandManager
+	METHOD b getCommands ()Ljava/util/Map;
```

Comment:
I prefer getRegisteredCommands or getManagedCommands. That's more opinion than anything though.